Source/WebCore/html/CheckboxInputType.h

```diff
@@ -57,6 +60,12 @@ class CheckboxInputType final : public BaseCheckableInputType {
     void didDispatchClick(Event&, const InputElementClickState&) final;
     bool matchesIndeterminatePseudoClass() const final;
     bool shouldAppearIndeterminate() const final;
+
+    void switchAnimationTimerFired();
+
+    Seconds m_switchAnimationStartTime { 0_s };
+    Seconds m_switchAnimationDuration;
```

Comment:
I don't think this needs to be a member variable.

Reply:
So we just call the theme method each time? That works. Progress does seem to store it though.